mgmt/rest/rest_v2_test.go

```diff
@@ -288,9 +288,10 @@ func TestV2Task(t *testing.T) {
 			body, err := ioutil.ReadAll(resp.Body)
 			So(err, ShouldBeNil)
 			So(
-				fmt.Sprintf(mock.ADD_TASK_RESPONSE, r.port),
+				string(body),
 				ShouldResemble,
-				string(body))
+				fmt.Sprintf(mock.ADD_TASK_RESPONSE, r.port),
+			)
```

Comment:
In this test file there are only changes related to keeping the pattern that an actual result should be at the first position:
```
So(actual-result, assertion, expected-result)
```